src/app/challenges/[slug]/ChallengeSubmit/CodeEditor.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
+
+import React, { useState } from "react";
+import Editor, { OnMount, type Monaco } from "@monaco-editor/react";
+import { CircularProgress, Box, Typography } from "@mui/material";
+import EditorTooltip from "@/components/EditorTooltip";
+
+import editorTheme from "@/theme/editorTheme";
+import Button from "@/components/Button";
+import { useParams } from "next/navigation";
+import SuccessAlert from "./SuccessAlert";
+
+enum TestStatus {
+  SUCCESS = "success",
+  ERROR = "error",
+  UNDEFIEND = "",
```

Comment:
couple instances of this in the file, nit: 
```suggestion
  UNDEFINED= "",
```